src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -37,53 +37,61 @@ const ViewConceptsHeader: React.FC<Props> = ({
         handleSwitchSourceClose
     ] = useAnchor();
 
+    const getTitleBasedOnContainerType = () => {
```

Comment:
it is better if we remove the ternary operator and use an if condition, code will look much cleaner & readable